src/components/api-guide/ApiGuide/DerivGuideContent/DerivGuideContent.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react"
 import { Link } from "react-router-dom";
 import ApiGuideTable from "../ApiGuideTable/ApiGuideTable";
 import CodeBlockSingleLanguage from "../../../global/CodeBlock/CodeBlockSingleLanguage";
+import { InlineCode } from "../../../global/InlineCode/InlineCode";
```

Comment:
```suggestion
import { InlineCode } from "global/InlineCode/InlineCode";
```
could you please check if you have config for the project with src directory set directly in order to put the file path starting from an src subfolder? or could you maybe check if this works?🙂

Reply:
Ah. this folder is actually not set in the root of the project. It is inside src > components > global. We have not a config set for this project to direct to a certain folder.